doc/nrf/doc_styleguide.rst

```diff
@@ -23,109 +22,63 @@ More information about RST is available in the `reStructuredText Primer`_.
+| Sending shell commands                            | Sending Shell Commands                            |
++---------------------------------------------------+---------------------------------------------------+
+| GATT Human Interface Device (HID) Service Client  | GATT Human Interface Device (HID) service client  |
++---------------------------------------------------+---------------------------------------------------+
 
 Table of contents
------------------
-
-If your page uses sections, add the ``.. contents::`` directive just under the page title:
-
-.. code-block:: none
-
-    1 .. _lib_aws_fota:
-    2
-    3 AWS FOTA
-    4 ########
-    5
-    6 .. contents::
-    7    :local:
-    8    :depth: 2
+=================
 
+If your page uses sections, add the ``.. contents::`` directive just under the page title.
 This will add a clickable table of contents at the top of the page.
 
-.. note::
-    Do not use table of contents if the page has a list of `Subpages`_.
-    To make the page hierarchy easier to navigate, using heading-based sections with the subpages section on one page is not allowed.
+For easy navigation, do not include a table of contents if the page has a list of subpages.
 
 Subpages
---------
+========
 
 Use the ``.. toctree::`` directive at the bottom of a page to list pages that are located further down in the hierarchy.
-For example, the :ref:`documentation` page has the following list of subpages, which includes this very page you are reading:
-
-.. code-block:: none
-
-   .. toctree::
-      :maxdepth: 2
-      :caption: Subpages:
+For example, the :ref:`documentation` page has a list of subpages, which includes this very page you are reading.
 
-      doc_build
-      doc_styleguide
-      doc_templates
-
-For clarity of structure, pages with the subpages section must not contain sections.
+For clarity of structure, pages with the subpages section must not contain heading-based sections or a table of contents.
 
 RST file formatting
 ===================
 
 For readability reasons, start every sentence on a new line in the source files and do not add line breaks within the sentence.
-In the output, consecutive lines without blank lines in between are combined into one paragraph.
+In the final output, consecutive lines without blank lines in between are combined into one paragraph.
 
-.. note:: For the conceptual documentation written in RST, you can have more than 80 characters per line.
-          The requirement for 80 characters per line applies only to the code documentation written in doxygen.
+For conceptual documentation written in RST, you can have more than 80 characters per line.
+The common limit of 80 characters per line applies only to the code documentation written in doxygen.
 
-The sentences must not end with a space.
+Do not end sentences with a space.
 Trim trailing spaces before committing your changes.
 
 Each RST file must end with one blank line.
```

Comment:
I would actually remove everything that follows this line up to Linking. The "would-remove" content is either covered in the Confluence style guide or can be covered there, and has more to do with guidelines we tech writers set rather than guidelines we inherited or baked into CI that could impede a PR from going in (like missing blank line at the end of the file). This could actually be a rule for this style guide: describe conventions that are linked to CI checks.